### ISSUE TITLE
Assimp Material nodes go wacko when multiple SceneFiles

### ISSUE THREAD

**Issue by mediadog** [mrvux/dx11-vvvv#296]
I found that when loading multiple .dae models using multiple SceneFile and Material nodes, two bad things happen: 1) some of the Material nodes will turn red, which ones depending on what order the SceneFile nodes are updated, and 2) the Texture Operation pins outputs a large number instead of something sensical like "Multiply", with the node turning red when you mouse over that pin.

The simple attaches patch shows these behaviors.

After the patch is loaded, first mouse over the left-most Texture Operation pin, it says Multiply.  Then mouse over the middle mode Operation pin, it says Multiply.  But the right node pin is nonsense and turns red.  Fool around with the reload pins, and various other nodes may turn grey or red.  Go back to the left Material node Operation pin, and it is now nonsense.

**Comment by mediadog** [comment]
Hmmm, github is not taking the zip file... trying again...  Nope!  WTF?  I'm trying to attach a .zip file, and it says "We don't support that file type." and then lists ZIP as one it does!

OK, trying an image of the patch...
![dx11assimpmaterialbug_2017 03 02-22 26 11](https://cloud.githubusercontent.com/assets/332032/23537035/4868daea-ff97-11e6-806c-1419ac786b77.png)

**Comment by mrvux** [comment]
Actually I can even reproduce while just reloading a single file/single material.
Seems like the loader grabs weird number for enum (hence no error is given anywhere since enum is cast to int directly).

I checked seems to come from the c++ loader side, so have to look that direction

**Comment by mrvux** [comment]
Yes looks like a bit, not sure if it comes from assimp itself or wrapper (nothing I can see from wrapper doing that, but upgrade to newer one is still in the plans at some point)

**Comment by mediadog** [comment]
My suspicion is it's Assimp itself; I bet the model has some bad elements (an export from Rhino, an iffy program in my book, where lots of things have Chinese character names).  It takes a very disciplined programmer to always, always check return values and properly handle try/catch exceptions.  I do of course! (;})
